CMakeLists.txt

```diff
@@ -91,69 +30,77 @@ endif(CMAKE_SIZEOF_VOID_P MATCHES "8")
 ## Specify additional locations of header files
 ## Your package locations should be listed before other locations
 # include_directories(include)
-include_directories(${catkin_INCLUDE_DIRS} ${VICON_SDK_INCLUDE_DIR})
+include_directories(include ${Boost_INCLUDE_DIRS} ${VICON_SDK_INCLUDE_DIR})
```

Comment:
I don't think we're actually using Boost -- try removing this and the `Boost_LIBRARIES` one too

Reply:
Removing Boost will cause build failure